docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -46,11 +46,7 @@
    "source": [
     "# some basic imports\n",
     "import numpy as np\n",
-    "import warnings\n",
-    "\n",
-    "with warnings.catch_warnings(action=\"ignore\", category=DeprecationWarning):\n",
-    "    # suppress warning about pyarrow as future required dependency\n",
-    "    import pandas as pd\n",
```

Comment:
oh was this issue with pandas fixed? 😮

Reply:
Yup, no I don't get an issue anymore, so we might as well remove it altogether.